type/browser/browser.js

```diff
@@ -13,6 +14,7 @@ import * as Assistant from "./assistant"
 
 export type Model = {
   version: Version,
+  mode: PerspectiveUI.Model,
```

Comment:
Browser module should not be aware of perspective UI at all perspective UI should embed browser instead IMO.